oap-server/server-storage-plugin/storage-iotdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/iotdb/query/IoTDBTopologyQueryDAO.java

```diff
@@ -254,7 +254,7 @@
                 RowRecord rowRecord = wrapper.next();
                 List<Field> fields = rowRecord.getFields();
                 Call.CallDetail call = new Call.CallDetail();
-                List<String> layerNames = Splitter.on(IoTDBClient.DOT)
+                List<String> layerNames = Splitter.on(IoTDBClient.DOT + "\"")
```

Comment:
What is the splitter in the database?

Reply:
Could you share some examples? I am a little not following yet.

Reply:
Is this index mechanism working everywhere? Be careful for trace/log's tags, or endpoint name. It is common to have `.` and `"` from there. I think IoTDB community should improve this, it is hard to choose and separate fields in multi values index.

Reply:
We did add the endpoint name and instance name because we are concerned about `too many candidates` for IoTDB. But in real cases, we may face performance issues when user wants to find an endpoint from 1m size data set.

Reply:
Anyway, we have known, IoTDB as storage is not used in the product mostly for now, because we didn't receive any report about this feature.